modules/notes/controllers/createNotes.js

```diff
@@ -0,0 +1,12 @@
+import { notesDB } from "../../../db/providers/notesProvider.js";
+import logger from "../../../utils/logger.js";
+
+export const postNotes = async (req, res) => {
+  try {
+    //   const { title, content, timestamps } = req.body;
+    const note = await notesDB.create(req.body);
```

Comment:
It's a hack. Separate all the fields that you need and send them, not all the BODY